tests/test_tokenize.py

```diff
@@ -146,9 +147,8 @@ def test_stream_error():
     'FULL OUTER JOIN',
     'NATURAL JOIN',
     'CROSS JOIN',
-    'STRAIGHT JOIN',
-    'INNER JOIN',
-    'LEFT INNER JOIN'])
+    'STRAIGHT_JOIN',
+    'INNER JOIN'])
```

Comment:
No connectors support `STRAIGHT JOIN`. It's a MySQL thing but it only allows the underscore version `STRAIGHT_JOIN` (which is included in the keyword list below). Also `LEFT INNER JOIN` is not a thing in any database - `{LEFT,RIGHT,FULL}`  can only be used with `OUTER` (which can be omitted because in which case it's implicitly outer).